app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java

```diff
@@ -141,10 +160,22 @@ public void onDrawerClosed(View drawerView) {
     public void setEventId(long eventId) {
         this.eventId = eventId;
         fragmentNavigator.setEventId(eventId);
-        binding.navView.getMenu().setGroupVisible(R.id.subMenu, true);
+
+        for (Integer itemId : drawerItems) {
+            binding.navView.getMenu().findItem(itemId).setVisible(true);
+        }
+
+        boolean isDeveloperModeEnabled = sharedPreferences.getBoolean(
+            getString(R.string.developer_mode_key), false);
+
+        if (isDeveloperModeEnabled) {
+            for (Integer itemId : drawerExtraItems) {
+                binding.navView.getMenu().findItem(itemId).setVisible(true);
+            }
+        }
 
         if (BuildConfig.HIDE_DRAWER_ITEMS) {
-            for (Integer itemId : drawerItems) {
+            for (Integer itemId : drawerExtraItems) {
```

Comment:
I don't get why we have `BuildConfig.HIDE_DRAWER_ITEMS` and 
`sharedPreferences.getBoolean( getString(R.string.developer_mode_key), false);`. Should one of them be removed?

Reply:
I'm not sure about the use of `HIDE_DRAWER_ITEMS`. It is set to true for the release version and false for the debug version. 
@iamareebjamal Should it be removed?

Reply:
Yes, remove it